source/includes/v2/_assets.md

```diff
@@ -30,33 +35,42 @@ var request = new AssetSearchRequest
 var response = client.AssetsV2.SearchAsync(request);
 ```
 
+REST API example:
+```json
+GET /api/v2/assets?blockchainId=bitcoin&symbol=BTC
+```
+
 ### Request
 
-name | type | description | example
----- | ---- | ----------- | -------
-`Id` | *optional*, *long* | ID of the asset to search | 100553
-`BlockchainId` | *optional*, *string* | Exact blockchain id to search | bitcoin-private
-`Symbol` | *optional*, *string* | Text to search in the asset symbol | BTC
-`TokenId` | *optional*, *string* | Text to search in the asset symbol | BTC
-`Address` | *optional*, *string* | Exact address to search | 0xC1701AbD559FC263829CA3917d03045F95b5224A
-`Accuracy` | *optional*, *int* | Exact accuracy to search | 8
-`Pagination` | *optional*, *[Pagination](#api-usage-pagination-request)* | |
+| name           | type                                                                  | description                                          | example                                    |
```

Comment:
rest/gRPC field names difference is not reflected here

Reply:
`+`